tests/shared/core/test_domain.py

```diff
@@ -859,6 +859,10 @@ def test_load_domain_with_entity_roles_groups():
     assert "origin" in domain.roles["GPE"]
     assert "destination" in domain.roles["GPE"]
 
+def test_load_domain_with_custom_slot():
+    domain = Domain.load("data/test_domains/custom_slot_domain.yml")
+
+    assert domain.slots is not None
```

Comment:
Shall we check that the limit param made it to the slot class too?

Reply:
oh yes sorry I never finished this test and forgot 🤦‍♀️

Reply:
updated